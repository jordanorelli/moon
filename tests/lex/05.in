
this whole line is one string value, y'all!

no_value:

key: a bare string is here

key: "string"

one: the first string; two: the second string
