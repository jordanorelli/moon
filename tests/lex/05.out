{t_string this whole line is one string value, y'all!}
{t_name no_value}
{t_object_separator :}
{t_name key}
{t_object_separator :}
{t_string a bare string is here}
{t_name key}
{t_object_separator :}
{t_string string}
{t_name one}
{t_object_separator :}
{t_string the first string}
{t_name two}
{t_object_separator :}
{t_string the second string}
